### PR TITLE
feat: add glossary

### LINKED ISSUE #7: Initiating a GLOSSARY.md file in this repo to have a document with definition of "DDD"-related terms
> :bulb: What do you think of initiating a GLOSSARY.md file in this repo to have a document with definition of "DDD"-related terms ?

Great idea. I made a [pull request](https://github.com/Mehdi-H/DDD-blue-book-collaborative-reading/pull/6) following your suggestion.

_Originally posted by @sofcalca in https://github.com/Mehdi-H/DDD-blue-book-collaborative-reading/discussions/3#discussioncomment-5687247_

### PR DESCRIPTION
**Pull request by sofcalca**
Added a GLOSSARY.md file to centralize definition of terms appearing in the book

### CODE REVIEW

File: GLOSSARY.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Glossary
+
+| Concept                       | Definition                                                                                                                                     |
+|-------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
+| APPLICATION COORDINATOR       |                                                                                                                                                |
```
Comment (Mehdi-H) [inline GLOSSARY.md:5]: I would write these concepts in lowercase

**Review by Mehdi-H — APPROVED** [review]
- I would precise the source of those definitions, 
- I would write the concepts in lowercase,
- and I would put a link toward this glossary in the README.md file to make it easily discoverable

Otherwise, everything's good for me !

File: GLOSSARY.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Glossary
```
[the remark this answers is not in the document]
Comment (sofcalca) [reply GLOSSARY.md:1]: I tried to copy-paste all terms that appear in upper case in the book and appear to be the _ubiquitous language_ of DDD :wink:

**Comment by Mehdi-H** [comment]
closes #6

**Comment by Mehdi-H** [comment]
Closes #7

**Pull request merged**